src/app/wizard/step/cluster/component.ts

```diff
@@ -307,8 +307,8 @@ export class ClusterStepComponent extends StepBase implements OnInit, ControlVal
   updateCNIPluginOptions() {
     if (
       this.controlValue(Controls.CNIPlugin) === CNIPlugin.Cilium &&
-      !!this.isKonnectivityEnabled &&
-      !!this.controlValue(Controls.Konnectivity)
+      this.isKonnectivityEnabled &&
+      this.controlValue(Controls.Konnectivity)
```

Comment:
Actually, `controlValue` returns `any` and can return `undefined` also. Here, it was ok :slightly_smiling_face:

Reply:
Good to know. Re-added :D